examples/icon.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019-2021 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+fn main() -> wry::Result<()> {
+    use wry::{
+      application::{
+        event::{Event, StartCause, WindowEvent},
+        event_loop::{ControlFlow, EventLoop},
+        window::WindowBuilder,
+      },
+      webview::WebViewBuilder,
+    };
+    use ico::*;
+
+    // use a window icon
+    let file = std::fs::File::open("<specify_the_icon_here>").unwrap();
+    let icon_dir = ico::IconDir::read(file).unwrap();
+    let image = icon_dir.entries()[0].decode().unwrap();
+    let rgba = image.rgba_data();
+    let icon = WindowIcon::from_rgba(rgba.to_vec(), 256, 256)?;

```

Comment:
In tauri, we use `include_bytes` macro and get the static icon in compile time.
It also doesn't require any dependency in this way.
Could you update to use this method?

Reply:
sure, I´ll update it later this day